tags/src/components/Tag.tsx

```diff
@@ -10,14 +10,19 @@ import {
 const Tag: FunctionComponent = () => {
   const { data, actions } = useFieldPlugin()
 
+  const contentValue = (data.content as { value: string[] }).value
```

Comment:
Hey @marcosh-ramos 🙌 

I found an issue with this implementation while using it in the Field Plugin Editor.
It happens that the plugin's value is initialized differently in the Field Plugin Editor.

While it starts as an empty string in the Visual Editor, in the Field Plugin Editor, it starts with `undefined` which make the plugin to break as per the image below.

![Screenshot 2025-02-11 at 08 08 27](https://github.com/user-attachments/assets/ffbe4146-3bbc-4c89-97e2-c65e1f171e69)

```suggestion
  const contentValue = (data.content as { value: string[] })?.value
```

Reply:
good catch! I'll commit your suggestion